src/util/timeConverterToMinSec.ts

```diff
@@ -0,0 +1,15 @@
+export function timeConverterToMinSec(time: string) : string {

```

Comment:
Have the input this function just be in millis please.

Reply:
Updated in subsequent commit!